Cargill/splinter-ui
# Add partial form to select nodes for circuit proposal

This PR adds a partial form to select nodes for a circuit proposal, and a form to add a new node to the splinter registry.

#### To test: 
1. Run:
```
docker-compose -f docker/docker-compose.yaml up --build
```
2. To see the alpha node UI go to: `localhost:3030`

3. Click on the `circuits` option in the left side menu, it will forward you to  `localhost:3030/circuits`:
<img width="1005" alt="Screen Shot 2020-06-11 at 8 15 48 AM" src="https://user-images.githubusercontent.com/14094978/84389783-333c1b80-abbc-11ea-95c6-a6bbeed42f82.png">


4. Click "Propose Circuit" button. You will be forward to the Propose Circuit Form. Test that you can select a node from the available nodes and remove a node from the selected nodes. 
<img width="1005" alt="Screen Shot 2020-06-11 at 8 15 17 AM" src="https://user-images.githubusercontent.com/14094978/84389803-39ca9300-abbc-11ea-9949-16253540c5c7.png">

5. The form is also partially responsive, in a mobile view:
<img width="825" alt="Screen Shot 2020-06-11 at 8 16 51 AM" src="https://user-images.githubusercontent.com/14094978/84389873-56ff6180-abbc-11ea-8265-1bee31671b2a.png">



P.S The other steps in the form are not implemented in this PR.
